conf/app.routes

```diff
@@ -2,4 +2,5 @@
 
 ->         /hmrc-frontend           hmrcfrontend.Routes
 GET        /account-check           uk.gov.hmrc.taxenrolmentassignmentfrontend.controllers.AccountCheckController.accountCheck(redirectUrl: String)
+GET        /pt-enrolment-check         uk.gov.hmrc.taxenrolmentassignmentfrontend.controllers.MultipleAccountsCheckController.personalTaxEnrolmentCheck(redirectUrl: String)
```

Comment:
This endpoint is for all the cases of multiple accounts not just for pt-enrolment check so the url should be more of the form /multiple-accounts/check